app/models/difficulty_level.rb

```diff
@@ -3,4 +3,5 @@ class DifficultyLevel < ActiveRecord::Base
   has_many :events, dependent: :nullify
 
   validates :title, presence: true
+  validates :color, format: /\A#[0-9a-fA-F]{6}\z/
```

Comment:
I'd really like to see that the validations are in a separate commit! Do you know how to do that?